source/extensions/filters/network/well_known_names.h

```diff
@@ -14,6 +14,8 @@ class NetworkFilterNameValues {
 public:
   // Client ssl auth filter
   const std::string ClientSslAuth = "envoy.client_ssl_auth";
+  // Dubbo proxy filter
+  const std::string DubboProxy = "envoy.filters.network.dubbo_proxy";
```

Comment:
Can we potentially pull all the filter out into the next PR? I don't think this part has any use here? Up to you.

Reply:
`extensions_build_config.bzl` depend on the ` "/source/extensions/filters/network/dubbo_proxy: config" ` target

Reply:
You can make that depends on hessian_utils for now and change to config later once all the PRs are in. One benefit of that is dubbo_proxy won't show up in `statically linked extensions` printed by Envoy during start up when it is not ready.

Reply:
OK，I'll give it a try